reference.md

```diff
@@ -18,46 +18,53 @@ _____
 - `ls -a` - list all files, including hidden files
 - `ls *.txt` - list all files that end with `.txt`
 
-**`cd`** change directory
+**`cd`** - change directory
 
-  `cd pathname` - takes you to the directory specified by `pathname`
-  
-  `cd ~` -  takes you to your home directory
+- `cd pathname` - takes you to the directory specified by `pathname`
+- `cd ~` -  takes you to your home directory
+- `cd ..` - takes you up one directory
+
+**`man`** - display the user manual
+
+**`history`** - displays the history list with line numbers, use `n` to limit the list
 
 ______
 ### Shell: Interacting with Files
 
-**`mkdir`** make a directory
+**`mkdir`** - make a directory
+
+**`cat`** - print to shell or send file or files to output 
 
-**`cat`** print to shell or send file or files to output 
+**`head`** - output first 10 lines of a file or files
 
-**`head`** output first 10 lines of a file or files
+**`tail`** - output last 10 lines of a file or files
 
-**`tail`** output last 10 lines of a file or files
+**`less`**
 
-**`mv`** rename or move a file or files. Syntax for renaming a file: `mv FILENAME NEWFILENAME`
+**`mv`** - rename or move a file or files. Syntax for renaming a file: `mv FILENAME NEWFILENAME`
 
-**`cp`** make a backup copy of a file or files. Syntax: `cp FILENAME NEWFILENAME`
+**`cp`** - make a backup copy of a file or files. Syntax: `cp FILENAME NEWFILENAME`
 
-**`>`** redirect output. Syntax with `cat`: `cat FILENAME1 FILENAME2 > NEWFILENAME`
+**`>`** - redirect output. Syntax with `cat`: `cat FILENAME1 FILENAME2 > NEWFILENAME`
 
-**`>>`** redirect output by appending to the filename specified. Syntax with `cat`: `cat FILENAME1 FILENAME2 >> NEWFILENAME`
+**`>>`** - redirect output by appending to the filename specified. Syntax with `cat`: `cat FILENAME1 FILENAME2 >> NEWFILENAME`
 
-**`rm`** remove a file or files. NB: *USE WITH EXTREME CAUTION!!!*
+**`rm`** - remove a file or files. NB: *USE WITH EXTREME CAUTION!!!*
 
-**`rmdir -r`** will delete a directory, even if it is not empty.
+**`rmdir -r`** - will delete a directory, even if it is not empty.
 
-**`rmdir -r-i`** will delete a directory, even if it is not empty, but will ask you to confirm each deletion.
+**`rmdir -r-i`** - will delete a directory, even if it is not empty, but will ask you to confirm each deletion.
 
+**`touch`** - update timestamp information on files
 
 ______
 ### Shell: Wildcards
 
-**`?`** a placeholder for one character or number
+**`?`** - a placeholder for one character or number
 
-**`*`** a placeholder for zero or more characters or numbers
+**`*`** - a placeholder for zero or more characters or numbers
 
-**`[]`** defines a class of characters
+**`[]`** - defines a class of characters
```

Comment:
Cosmetics again. Please don't merge the changes between `history` & this one, except for the L58 with `touch`.

Reply:
@katrinleinweber I would appreciate knowing why you think the Maintainers for this lesson shouldn't merge the cosmetic changes for the reference file?

Reply:
Same reason [as above](https://github.com/LibraryCarpentry/lc-shell/pull/61#discussion_r294977017).

Reply:
@katrinleinweber I am trying to understand your point of view about cosmetic changes so that I can add that to the Maintainer discussion.  Just telling me they shouldn't be merged doesn't tell me WHY you don't think they should be merged.

Reply:
The comment I linked to includes a _why_: "… content and style changes … mashed into 1 commit … makes researching the reasons for particular changes harder in the future."

Reply:
I'm sorry I didn't understand but it sounded like you were saying that consistency changes shouldn't be done at all, which is what I was trying to understand.  I already said I would address the other point about them being mashed into 1 pull request.  I'm new to being a maintainer so I don't have all the history that everyone else has.